Fix CI (DONE for master, TODO for PRs)

CI is not here to be ignored. If it fails, fix it. I'm a bit tired of being spammed by it.

@Drup It seems it's during the documentation generation. I resolved two problems but I have the following error:
```
ODOC_WIKI_SUBPROJECT="client" \
    eliomdoc -client \
    -ppx -package yojson,calendar,ocsigen-widgets.client \
     -intro doc/indexdoc.client -colorize-code -stars -sort \
     -i  -g odoc_wiki.cma \
     -d doc/client/wiki \
     -I _client/src/ \
     src/os_tips.eliomi src/os_uploader.eliomi src/os_tools.eliomi src/os_handlers.eliomi \
     src/os_date.eliomi src/os_push_notifications.eliomi src/os_userbox.eliomi \
     src/os_services.eliomi
     src/os_email.eliomi src/os_comet.eliomi src/os_notif.eliomi src/os_lib.eliomi \
     src/os_user_proxy.eliomi src/os_session.eliomi src/os_current_user.eliomi src/os_page.eliomi \
     src/os_user.eliomi src/os_msg.eliomi src/os_request_cache.eliomi src/os_view.eliomi

Fatal error: exception Failure("Unknown extension for file odoc_wiki.cma.")
Makefile:240: recipe for target 'doc' failed
```
Is it a problem with wikidoc?

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,0 +1,11 @@
+language: c
+sudo: required
+install: wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.travis-opam.sh
+script: bash -ex .travis-opam.sh
+env:
+  - OCAML_VERSION=4.02 PINS="ocsigenserver:https://github.com/ocsigen/ocsigenserver.git reactiveData:https://github.com/ocsigen/reactiveData.git eliom:https://github.com/ocsigen/eliom.git ocsigen-toolkit:https://github.com/ocsigen/ocsigen-toolkit.git" PACKAGE=ocsigen-start
+  - OCAML_VERSION=4.03 PINS="ocsigenserver:https://github.com/ocsigen/ocsigenserver.git reactiveData:https://github.com/ocsigen/reactiveData.git eliom:https://github.com/ocsigen/eliom.git ocsigen-toolkit:https://github.com/ocsigen/ocsigen-toolkit.git" PACKAGE=ocsigen-start
+  - OCAML_VERSION=4.04 PINS="ocsigenserver:https://github.com/ocsigen/ocsigenserver.git reactiveData:https://github.com/ocsigen/reactiveData.git eliom:https://github.com/ocsigen/eliom.git ocsigen-toolkit:https://github.com/ocsigen/ocsigen-toolkit.git" PACKAGE=ocsigen-start
+os:
+  - linux
+  - osx

--- a/README.md
+++ b/README.md
@@ -1,4 +1,7 @@
-#Ocsigen-start
+# Ocsigen-start [![Travis Status][travis-img]][travis]
+
+[travis]:      https://travis-ci.org/ocsigen/ocsigen-start/branches
+[travis-img]:  https://img.shields.io/travis/ocsigen/ocsigen-start/master.svg?label=travis
 
 1. [Getting started](#getting-started)
 

